doc/user-guide/Features.md

```diff
@@ -78,11 +78,11 @@ On more information on how to set up this feature, see [dynamic domains configur
 
 In order to build client applications, the MongooseIM team recommends the following libraries:
 
-| |XMPP|REST API|
-| ------------- | ------------- | ------------- |
-|iOS|[XMPPframework](https://github.com/robbiehanson/XMPPFramework), Objective-C|[Jayme](https://github.com/inaka/Jayme), Swift|
-|Android|[Smack](https://github.com/igniterealtime/Smack), Java|[Retrofit](https://github.com/square/retrofit), Java|
-|Web|[Stanza.io](https://github.com/otalk/stanza.io)/[Strophe.js](https://github.com/strophe/strophejs), JavaScript||
+| |XMPP|GraphQL API|REST API|
+| ------------- | ------------- | ------------- | ------------- |
+|iOS|[XMPPframework](https://github.com/robbiehanson/XMPPFramework), Objective-C|[Apollo](https://github.com/apollographql/apollo-ios), Swift|[Jayme](https://github.com/inaka/Jayme), Swift|
```

Comment:
I am not sure if we are ready to recommend Apollo because we haven't checked if ti works with MongooseIM, right? Maybe we need to do a separate research on GraphQL client libraries. My quick investigation shows that subscriptions are handled in a different way - they are using the WS transport, while we are using SSE... So I think that finding a fully compatible client library or adjusting MongooseIM to support one would be a separate task.